package.json

```diff
@@ -57,6 +61,7 @@
     "eslint-plugin-react": "^5.0.0",
     "live-server": "^0.8.1",
     "node-libs-browser": "^0.5.2",
+    "npm-watch": "^0.1.9",
```

Comment:
how does this watch functionality differ from what we already have in the webpack-dev-server?